### ISSUE TITLE
Bug: edge device node properties disappear from UI

### ISSUE THREAD

**Issue by kmbarton423** [open-horizon/anax#3352]
### Describe the bug.

Registered ubuntu edge device:
   hzn register --policy=kmbnode.json

   cat kmbnode.json
   {"properties": [
            { "name": "kmbnode", "value": "indeed"}
   ] }

From the UI, selected the edge device and confirmed "kmbnode" property was listed.
Added an additional property "duffus".
After a few minutes, selected the edge device again and found neither of these properties were listed.
Based on listpolicy for the node, appears properties were moved into deployment:properties (see attached).

Very confusing that these properties are no longer presented in the UI
[listPolicies.txt](https://github.com/open-horizon/anax/files/9013495/listPolicies.txt)
.

### Describe the steps to reproduce the behavior.

Steps to recreate are in the bug description.

### Expected behavior.

Expected node properties to not disappear.

### Operating Environment

Working with IEAM 4.4 development level 2.30.0-917

**Comment by johnwalicki** [comment]
The IEAM Web UI is not part of the OpenHorizon project.  If this is a bug in the IEAM user experience, there is an internal IBM GitHub Enterprise org to track it.

**Comment by dlarson04** [comment]
@johnwalicki   you are correct that the UI is not in this project... but it is the anax code that has changed and is going to cause a problem for the IEAM product

**Comment by linggao** [comment]
The PR [open-horizon/anax#3384](https://github.com/open-horizon/anax/issues/3384) just fixed the problem for node policy conversion from v1 to v2. The problem for UI still exists. An issue has been opened in the UI repository.

**Comment by kmbarton423** [comment]
Verified fix good on build 928.
Added comment to UI issue about current behavior observations.
